pypeit/spectrographs/gemini_gnirs.py

```diff
@@ -328,6 +333,8 @@ def orders(self):
 
     @property
     def spec_min_max(self):
+        # TODO: Why aren't these numbers in fraction of the detector
```

Comment:
Good question, although I don't think you can bin near-IR detectors.

Reply:
Right.  It's mostly just a comment about consistency, given that most (all?) of the other positions are given as relative to the detector size (for all echelles, regardless of their wavelength range).  Happy to leave it or change it and propagate the change for all spectrographs.

Reply:
They probably should be converted into fraction of the detector. We do use this for X-shooter VIS, which can be binned. Worth checking what we are doing there.

Reply:
Punting to a different PR.